### DISCUSSION TITLE
Repeating log message every 15 seconds

### DISCUSSION THREAD

**Discussion by cuiper** [netbox-community/netbox-docker#1047] [Getting Help]
at the moment netbox containers are up this comes. every 15 sec. Is this normal?

netbox-docker-netbox-1               | 127.0.0.1 - - [15/Aug/2023:14:31:01 +0000] "GET /api/ HTTP/1.1" 200 510 "-" "curl/7.81.0"
netbox-docker-netbox-1               | 127.0.0.1 - - [15/Aug/2023:14:31:16 +0000] "GET /api/ HTTP/1.1" 200 510 "-" "curl/7.81.0"
netbox-docker-netbox-1               | 127.0.0.1 - - [15/Aug/2023:14:31:32 +0000] "GET /api/ HTTP/1.1" 200 510 "-" "curl/7.81.0"
netbox-docker-netbox-1               | 127.0.0.1 - - [15/Aug/2023:14:31:47 +0000] "GET /api/ HTTP/1.1" 200 510 "-" "curl/7.81.0"
netbox-docker-netbox-1               | 127.0.0.1 - - [15/Aug/2023:14:32:02 +0000] "GET /api/ HTTP/1.1" 200 510 "-" "curl/7.81.0"
netbox-docker-netbox-1               | 127.0.0.1 - - [15/Aug/2023:14:32:17 +0000] "GET /api/ HTTP/1.1" 200 510 "-" "curl/7.81.0"
netbox-docker-netbox-1               | 127.0.0.1 - - [15/Aug/2023:14:32:32 +0000] "GET /api/ HTTP/1.1" 200 510 "-" "curl/7.81.0"

**Comment by tobiasge** [comment]
That's expected because the health checks are running every 15 seconds.
